azure/packages/azure-client/api-extractor.json

```diff
@@ -1,12 +1,4 @@
 {
-  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-  "extends": "@fluidframework/build-common/api-extractor-common-report.json",
-  "apiReport": {
-    "enabled": true,
-    "reportFolder": "<projectFolder>/../../../api-report/"
-  },
-  "docModel": {
-    "enabled": true,
-    "apiJsonFilePath": "<projectFolder>/../../_api-extractor-temp/doc-models/<unscopedPackageName>.api.json"
-  }
+    "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+    "extends": "../../api-extractor.json"
```

Comment:
Since we don't use this pattern elsewhere (a root config that we inherit from), I'd prefer to just duplicate the config in each project.

Reply:
Okay. Can do.